runner.go

```diff
@@ -107,6 +109,19 @@ func Run(options ...RunOption) {
 		return
 	}
 
+	if describeTasks {
+		var outputString string
+		for _, task := range tasks {
+			if len(task.Description) > 0 {
```

Comment:
I think we could clean up this conditional. Seems like we always know we want to display the task and _sometimes_ want to display the description. Could we compose this in such a way that we don't need an if/else?

Reply:
I wanted to conditionally include some punctuation and go doesn't have a ternary operator, unless I missed something. I removed the punctuation and conditional formatting.